index.html

```diff
@@ -0,0 +1,336 @@
+        <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+              alt="tiwtter"></a></li>
+        <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+      </ul>
+      <img src="./assets/images/Group.png" alt="">
+    </div>
+  </section>
+  <section class="desk-first-page">
+    <div class="desk-headline">
+      <h1 class="desk-hey">Hey There I'm Solomon</h1>
+      <h2 class="desk-softwaredev"> I am software developer</h2>
+      <p class="desk-headline-p">I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <ul class="socila-link">
+      <li class="social"><a href="https://github.com/solog0039"><img src="./assets/socialmedia/github.png"
+            alt="Github"></a></li>
+      <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img
+            src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+      <li class="social"><a href="https://twitter.com/Solomon57320119"><img src="./assets/socialmedia/tiwtter.png"
+            alt="tiwtter"></a></li>
+      <li class="social"><a href="##"><img src="./assets/socialmedia/m.png" alt="M"></a></li>
+    </ul>
+    <img src="./assets/images/Gourp1-desktop.png" alt="Image in the first page">
+  </section>
+  <section class="cards">
+    <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+    </div>
+    <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
+
+    <div class="card-1">
+      <h2 class="multi-story">Multi-Post Stories</h2>
+      <p class="p21">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="skills">
+        <li class="css-1 button">css</li>
+        <li class="skill button">html</li>
+        <li class="skill button">bootstrap</li>
+        <li class="skill button">Ruby</li>
+      </ul>
+      <a class="see-project-1 button" href="">See Project</a>
+    </div>
+    
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
```

Comment:
- Hi @solog0039 , you have done a geat job implementing the desktop version on your portfolio.

- I would like to kindly suggest that it's important for the `see project` button to appear on each card when you hover on it. Currently, there's no `see-project` button, except for the ones on the mobile version.

Can we fix it.


Here is what you have, although, all you need to do is display the see-project button when you hover 👇🏽 

| Your project card |
| :---:   | 
|<img width="1428" alt="Screenshot 2023-05-04 at 11 45 17" src="https://user-images.githubusercontent.com/99246359/236271148-c0fc71f8-4cac-4839-b41f-22e192d1809a.png">  |

Here is what you should do instead 👇🏽 

| Default Mood, no button shows  |This should appear when you hover, it includes the button |
| :---:   | :---: | 
| <img width="238" alt="Screenshot 2023-05-04 at 11 38 24" src="https://user-images.githubusercontent.com/99246359/236270292-6e33ac98-524d-4cee-8df9-228fd1198302.png"> |<img width="240" alt="Screenshot 2023-05-04 at 11 38 13" src="https://user-images.githubusercontent.com/99246359/236270331-6864f7a3-6ba5-4234-aaf9-a59ad605fb79.png">  |

- I'd appreciate if you can implement this changes on all project cards. 🌸 

What do you think?? 🤗